src/current/v23.2/set-up-physical-cluster-replication.md

```diff
@@ -0,0 +1,309 @@
+--max-sql-memory=.25 \
+--background \
+--config-profile replication-source
+~~~
+
+### Connect to the primary cluster system interface
+
+Connect to your primary cluster's system interface using [`cockroach sql`]({% link {{ page.version.version }}/cockroach-sql.md %}).
+
+1. To connect to the system interface, pass the `options=-ccluster=system` parameter in the URL:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach sql --url \
+    "postgresql://root@{your IP or hostname}:26257/?options=-ccluster=system&sslmode=verify-full" \
+    --certs-dir "certs"
+    ~~~
+
+    You will find that the prompt will include `system` when you have successfully connected.
+
+    {{site.data.alerts.callout_info}}
+    You should only use the system interface for administration. To work with databases, tables, or workloads, use the application virtual cluster{% comment %}({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}){% endcomment %}.
+    {{site.data.alerts.end}}
+
+1. Add your cluster organization and {{ site.data.products.enterprise }} license to the cluster:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET CLUSTER SETTING cluster.organization = 'your organization';
+    ~~~
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET CLUSTER SETTING enterprise.license = 'your enterprise license';
+    ~~~
+
+1. Confirm the status of your virtual cluster: {% comment %}Link to SQL ref page here{% endcomment %}
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SHOW VIRTUAL CLUSTERS;
+    ~~~
+
+    The output will include the `system` interface and the `application` virtual cluster:
+
+    ~~~
+    id |    name     | data_state | service_mode
+    ---+-------------+------------+---------------
+    1  | system      | ready      | shared
+    2  | template    | ready      | none
+    3  | application | ready      | shared
+    (3 rows)
+    ~~~
+
+### Create a replication user and password
+
+The standby cluster requires a user profile on the primary cluster to connect to.
+
+1. From the primary's system interface SQL shell, create a user and password:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE USER {your username} WITH LOGIN PASSWORD '{your password}';
+    ~~~
+
+1. Grant the [`REPLICATION` system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#supported-privileges) to your user: {% comment %}Link to detail on replication system privilege{% endcomment %}
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    GRANT SYSTEM REPLICATION TO {your username};
+    ~~~
+
+    If you need to change the password later, refer to [`ALTER USER`]({% link {{ page.version.version }}/alter-user.md %}).
+
+### Connect to the primary virtual cluster (optional)
+
+1. If you would like to add a workload to the primary's application virtual cluster, open a new terminal window, and connect with:
```

Comment:
Not sure on the "add a workload" language. I think the value here is showing the user how to connect to the application tenant. They can use the ccluster=application option as you did below. But, if they've followed the instructions to this point, they can also elide that option as the application cluster will be the default.

Reply:
I have adjusted this slightly so that users know that the `ccluster=application` param will connect them to the application VC.